### PR TITLE
Add my profile

### PR DESCRIPTION
**Pull request by SebasttianDaza**
## Changes proposed

In this PR I just added my profile.

## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2004"></a>

### CODE REVIEW

File: public/data/SebasttianDaza.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Sebastian Daza",
+  "type": "personal",
+  "bio": "Software Engineer, FullStack Developer and Open source enthusiast.",
+  "avatar": "https://avatars.githubusercontent.com/u/81209181?v=4",
```
Comment (Panquesito7) [inline public/data/SebasttianDaza.json:5]: This is a shorter way of using your image link. Plus, it updates automatically.
```suggestion
  "avatar": "https://github.com/SebasttianDaza.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/SebasttianDaza.json (hunk 1 above)
Comment (SebasttianDaza) [reply public/data/SebasttianDaza.json:5]: Good to know, Thanks

**Review by imran1509 — COMMENTED** [review]
Looks good to me.
Really great.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Pull request merged**